Ginger/GingerCore/DataSource/AccessDataSource.cs

```diff
@@ -34,9 +34,9 @@ public class AccessDataSource : DataSourceBase
     {
         private static readonly Object thisObj = new object();
 
-        private string GetConnectionString(string sFilePath, string sMode = "Write")
+        private string GetConnectionString(string sMode = "Write")
         {
-            FileFullPath = amdocs.ginger.GingerCoreNET.WorkSpace.Instance.SolutionRepository.ConvertSolutionRelativePath(sFilePath);
+            //FileFullPath = amdocs.ginger.GingerCoreNET.WorkSpace.Instance.SolutionRepository.ConvertSolutionRelativePath(sFilePath);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4855725)